config/routes.rb

```diff
@@ -172,6 +173,10 @@
           # See the above :admin_settings routes, outside of :communities resource
           get :settings,       to: redirect("/admin/settings")
           put :update_settings # PUT request, no redirect
+          get "getting_started_guide(/*all)", to: redirect {|p, req|
+            sub_path = p[:all] == "" ? "" : "/#{p[:all]}"
+            "#{p[:locale]}/admin/getting_started_guide#{sub_path}"
+          }
```

Comment:
I'm not sure if this is worth the trouble since we just launched this feature. 
(Or if there's more elegant way to do this kind of redirection including sub paths.)